packager/app/vlog_flags.h

```diff
@@ -7,9 +7,10 @@
 #ifndef APP_VLOG_FLAGS_H_
 #define APP_VLOG_FLAGS_H_
 
-#include <gflags/gflags.h>
+#include <absl/flags/declare.h>
+#include <absl/flags/flag.h>
 
-DECLARE_int32(v);
-DECLARE_string(vmodule);
+// ABSL_DECLARE_FLAG(int32_t, v);
```

Comment:
Still WIP?

Reply:
Going to kill this now, and circle back on logging after using abseil log.